docs/network/evm-nodes.md

```diff
@@ -255,13 +254,23 @@ The node throws an error if you try to run it in a mode that it is not configure
 
 ### From an existing Etherlink Smart Rollup node
 
-1. Download [an Etherlink Smart Rollup node snapshot](https://snapshots.tzinit.org), and use the `octez-smart-rollup-node` binary to import it in a temporary directory.
-The following examples use `<SR_OBSERVER_DATA_DIR>` as the location of this temporary directory.
+You can use the Etherlink Smart Rollup node to initialize a data directory for the EVM node without having to actually start the Smart Rollup node.
+The `octez-smart-rollup-node` binary sets up a data directory that the EVM node can use as a starting point.
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
+The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/tezos/tezos/-/releases.
+
+1. Download an Etherlink Smart Rollup node snapshot for the appropriate Etherlink network:
+
+   - For Mainnet, see https://snapshots.tzinit.org/etherlink-mainnet
+   - For Testnet, see https://snapshots.tzinit.org/etherlink-ghostnet
+
+1. Use the `octez-smart-rollup-node` binary to import the snapshot into a temporary directory.
```

Comment:
Same remark, we also support downloading snapshots from URL.

Reply:
Reworked to make the URL method the primary method.